automation/verify_attestation_td_guest.sh

```diff
@@ -14,8 +14,6 @@ setup_intel_tiber_trust_service(){
                 export http_proxy="$http_proxy"
                 export https_proxy="$https_proxy"
         fi
-        cd $ATTESTATION_DIR
```

Comment:
Not related to this line but do you want to change the function name as something like `setup_proxy_inside_guest()` as this is not doing anything related to tiber service now? and move the code that runs the `trustauthority-cli version` to the next function `verify_intel_tiber_trust_service()`?